app/controllers/posts_controller.rb

```diff
@@ -0,0 +1,72 @@
+class PostsController < ApplicationController
+  before_action :set_post, only: %i[show edit update destroy]
+
+  # GET /posts or /posts.json
+  def index
```

Comment:
Requirement from [Odin page](https://www.theodinproject.com/paths/full-stack-ruby-on-rails/courses/ruby-on-rails/lessons/authentication):
_9.- Test it out – sign out and go to the index page. You should see a list of the posts but no author names. Sign in and the author names should appear. Your secrets are safe!_

I can't validate this feature when I try to go to your home page, the app keeps in the login page. Kindly fix this issue.